### PR TITLE
MGMT-9914: Integration of the ODF LVM operator with assisted installer

### PR DESCRIPTION
**Pull request by dvachhan**
This PR depends on [openshift/assisted-service#3921](https://github.com/openshift/assisted-service/pull/3921)

### CODE REVIEW

**Comment by osherdp** [comment]
/uncc
@dvachhan please cc me again when you want me to review :)

**Comment by osherdp** [comment]
@dvachhan can you add a job so that we can see how it works?
for example: https://github.com/openshift/release/blob/b21f28ae0b8d8a04b296c0c973fab00280a07ffa/ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml#L260-L267

File: src/consts/olm_operators.py [hunk 1]
```diff
@@ -79,6 +80,11 @@ def values(cls) -> dict:
                 worker_count=4,
             ),
             OperatorType.LSO: cls.get_resource_dict(),
+            OperatorType.LVM: cls.get_resource_dict(
+                master_memory=1200,
```
Comment (osherdp) [inline src/consts/olm_operators.py:84]: just so I understand, those are additional resources that are required for the operator right?

**Review by osherdp — CHANGES_REQUESTED** [review]

**Review by gamli75 — APPROVED** [review]

**Review by adriengentil — APPROVED** [review]

**Pull request merged**